server/src/routes/rootRouter.js

```diff
@@ -2,10 +2,12 @@ import express from "express";
 import userSessionsRouter from "./api/v1/userSessionsRouter.js";
 import usersRouter from "./api/v1/usersRouter.js";
 import clientRouter from "./clientRouter.js";
+import indexRouter from "./api/v1/indexRouter.js"
```

Comment:
We want to update this to reflect the actual resource of `trails`, the resource is not called `index`